Haroon Shahid-Batch03 OOP Activity/Hard/HardQuestion1.cs

```diff
@@ -0,0 +1,40 @@
+using System;
```

Comment:
Explain the concept theoretically in two or three statements first before directly jumping to code. 
For example: Abstraction
What explain what is abstraction then give code example